packages/ra-ui-materialui/src/input/SelectArrayInput.tsx

```diff
@@ -247,6 +251,24 @@ export const SelectArrayInput = (props: SelectArrayInputProps) => {
         );
     }
 
+    const labelPlaceholder = translateLabel({
+        label,
+        resource,
+        source,
+    });
+
+    const outlinedInputProps =
+        variant === 'outlined'
+            ? {
+                  input: (
+                      <OutlinedInput
+                          id="select-multiple-chip"
+                          label={labelPlaceholder}
```

Comment:
```suggestion
                          label={
                              <FieldTitle
                                  label={label}
                                  source={source}
                                  resource={resource}
                                  isRequired={isRequired}
                              />
                          }
```

Otherwise you disable the feature that puts a `'*'` at the end of the label when the field is required